idd/Energy+.idd.in

```diff
@@ -16057,6 +16064,372 @@ SurfaceProperty:OtherSideConditionsModel,
       \note  in contact with GroundDomain objects
       \default GapConvectionRadiation
 
+Foundation:Kiva,
+       \memo Refined definition of the foundation surface construction used to
+       \memo infrom two-diemensional heat transfer calculated using the Kiva
+       \memo ground heat transfer methodology.
+       \extensible:4
+  A1,  \field Name
+       \required-field
+       \type alpha
+       \reference OutFaceEnvNames
+  A2,  \field Interior Horizontal Insulation Material Name
+       \type object-list
+       \object-list MaterialName
+  N1,  \field Interior Horizontal Insulation Depth
+       \note Distance from the slab bottom to the top of interior horizontal
+       \note insulation
+       \units m
+       \type real
+       \minimum 0.0
+       \default 0.0
+  N2,  \field Interior Horizontal Insulation Width
+       \note Extent of insulation as measured from the wall interior
+       \units m
+       \type real
+       \minimum> 0.0
```

Comment:
If a material is assigned, a zero value would imply no insulation. If a material is not assigned, a zero value is implicit. Do you think allowing zero would be better? I already have an error if a value (including zero) is defined here and no material is defined.